src/pages/learn/[id].tsx

```diff
@@ -1155,23 +1167,23 @@ const StStudyLogContainer = styled.div`
   flex-direction: column;
 `;
 
-const StStudyLogTabContainer = styled.div`
+const StStudyLogTabList = styled.ul`
   display: flex;
   align-items: center;
-  gap: 1.6rem;
   margin-bottom: 2rem;
-
-  .divider {
-    width: 0.2rem;
-    height: 1.6rem;
-    background-color: ${COLOR.GRAY_10};
-  }
 `;
 
-const StStudyLogTab = styled.h2<{ isActive: boolean }>`
+const StStudyLogTab = styled.li<{ isActive: boolean }>`
   color: ${({ isActive }) => (isActive ? COLOR.BLACK : COLOR.GRAY_30)};
   ${FONT_STYLES.SB_24_HEADLINE};
   cursor: pointer;
+
+  &:first-child:after {
+    content: '|';
```

Comment:
```suggestion
  &:not(:last-child):after {
    content: '|';
```
<img width="408" alt="image" src="https://user-images.githubusercontent.com/63948884/216281374-9497a90d-5321-4928-b3bf-22723398c5c4.png">

p4;
사실 탭이 더 늘어날 일은 없겠지만 재사용성을 고려해서 이런식으로 css 선택자를 구성하는 것은 어떨까요? 

그리고 복습하기 페이지랑 이 부분 모두 탭이 사용되고 있는데 탭 컴포넌트를 따로 분리하는 것도 좋을 것 같다는 생각이 드네요!

Reply:
오오 남겨주신 스타일링 방법 좋아요 반영하겠습니다 🙌🏻
탭 공통 컴포넌트로 만드는 것도 찬성합니다 나중에 담당자 정해봐용 ~!